app/controllers/idv/phone_controller.rb

```diff
@@ -61,8 +61,7 @@ def redirect_to_next_step
       if phone_confirmation_required?
         if VendorStatus.new.all_phone_vendor_outage?
           redirect_to vendor_outage_path(from: :idv_phone)
-        elsif step.otp_delivery_preference_missing?
-          redirect_to idv_otp_delivery_method_url
+        #   redirect_to idv_otp_delivery_method_url
```

Comment:
This comment is not needed anymore. Also you can delete the otp_delivery_preference_missing? method here https://github.com/18F/identity-idp/blob/main/app/services/idv/phone_step.rb#L47

Reply:
Removed both.